automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingApiSpec.scala

```diff
@@ -19,6 +21,7 @@ import scala.util.Try
 class BillingApiSpec extends FreeSpec with BillingFixtures with MethodFixtures with Matchers with Eventually
   with TestReporterFixture with LazyLogging {
 
+  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(1, Minutes)), interval = scaled(Span(20, Seconds)))
```

Comment:
default timeout is 150 millisec, need a longer timeout to verify billing account is associated with the google project.